### PR TITLE
1050: Move changeLogo.sh to /usr/bin

### PR DESCRIPTION
**Pull request by lxwinspur**
Fixes https://github.com/ibm-openbmc/dev/issues/3629

### CODE REVIEW

**Comment by gtmills** [comment]
add to approvelist

**Comment by gtmills** [comment]
544618 is the internal defect

**Comment by geissonator** [comment]
We got an internal failure building this:
```
ERROR: webui-vue-1.0+gitAUTOINC+b661b9be80-r1 do_package: QA Issue: webui-vue: Files/directories were installed but not shipped in any package:
  /usr/bin
Please set FILES such that these items are packaged. Alternatively if they are unneeded, avoid installing them or delete them within do_install.
```

Please look at this and be sure to add a "Tested:" to the commit msg indicating what testing was done (basically verify it compiles and script is in expected location)

File: meta-ibm/recipes-phosphor/webui/webui-vue_%.bbappend [hunk 1]
```diff
@@ -23,5 +23,5 @@ do_install:append:p10bmc() {
     install -m 0755 ${WORKDIR}/inspur-login-logo.svg.gz ${D}${datadir}/www/img
     install -m 0755 ${WORKDIR}/inspur-logo-header.svg.gz ${D}${datadir}/www/img
     install -m 0755 ${WORKDIR}/blankLogo.svg.gz ${D}${datadir}/www/img
-    install -m 0755 ${WORKDIR}/changeLogo.sh ${D}${datadir}/www
+    install -m 0755 ${WORKDIR}/changeLogo.sh ${D}${bindir}
```
Comment (anoo1) [inline meta-ibm/recipes-phosphor/webui/webui-vue_%.bbappend:26]: Need to create the dir to fix the compile error that Andrew mentioned:

```
+    install -d ${D}${bindir}
+    install -m 0755 ${WORKDIR}/changeLogo.sh ${D}${bindir}
```

After building, verify that the change-logo.service file has the expected line change, location:
```
tmp/work/p10bmc-openbmc-linux-gnueabi/obmc-phosphor-image/1.0-r0/rootfs/lib/systemd/system/change-logo.service
```

And that the script exists under this path:
```
tmp/work/p10bmc-openbmc-linux-gnueabi/obmc-phosphor-image/1.0-r0/rootfs/usr/bin/changeLogo.sh
```

Probably it'd be good too to flash the image to a system to verify the logo script runs as expected.

Document your results in the commit message under the "Tested:" tag as Andrew requested.

**Review by anoo1 — CHANGES_REQUESTED** [review]

File: meta-ibm/recipes-phosphor/webui/webui-vue_%.bbappend (hunk 1 above)
Comment (lxwinspur) [reply meta-ibm/recipes-phosphor/webui/webui-vue_%.bbappend:27]: Thanks @anoo1 @geissonator 
Updated

**Review by anoo1 — APPROVED** [review]

**Comment by gtmills** [comment]
@lxwinspur Could you push this changeLogo stuff to upstream meta-ibm ? Lower priority but would be good to not have to carry..

**Comment by gtmills** [comment]
Reed pulled this into our GitHub Enterprise and merged it. Closing this and the issue. Thanks for the help here!

**Pull request closed without merging**